### PR TITLE
Add v3 planning doc

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

**Comment by jmeas** [comment]
I think this is a good start. Maybe we should hold off on publishing anything like this, though. It seems like some of the things are still up in the air.

**Comment by samccone** [comment]
everything that we were semi firm on got the general :+1: on the call that we had .. this post is just an expanded version of the call notes

**Comment by samccone** [comment]
@jmeas updated

**Comment by jmeas** [comment]
:+1:

Last thing I would update is the title. "Marionette v3 -- The Plan" makes me think that this is the definitive plan, but this is just notes on our discussions of the plan. Maybe something like:

> Version 3 Discussions

or something. To make it more official you could call it meeting notes.

**Comment by jdaudier** [comment]
I think you meant to capitalize Marionette in the 1st sen.

File: source/_drafts/v3-planning.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - plans
+  - v3
+---
+
+This past friday the Marionette Team got together and did some v3 planning for the next release of marionette.
+The focus on of our conversation was on looking into adding the missing application layer pieces that are very much needed within Marionette.
+Over the past few years there has been a community effort to establish patterns to allow people to build large applications in Marionette.
+We are exploring ways that we can codify the concepts into the core.
+
+[@jmeas](https://github.com/jmeas/) has done some great work in this area via the creation of a library called [base-router](https://github.com/jmeas/backbone.base-router). We want to take this work and build ontop of it to create a router that has some new concepts such as state based routing, and named routes. There will most likely be more features, inspired both from the great ember router and angular ui router.
```
Comment (jdaudier) [inline source/_drafts/v3-planning.md]: OLD: build ontop of it
NEW: build on top of it

**Comment by thejameskyle** [comment]
:+1:

**Comment by jdaudier** [comment]
At the very end:
Note that these are early discussions and the final result may be different than what is listed above.

File: source/_drafts/v3-planning.md (hunk 1 above)
Comment (megawac) [inline source/_drafts/v3-planning.md]: Just got around to reading this

> inspired both from the great ember router and angular ui router.

inspired by the great router implementations of Ember and angular-ui.

Probably should also link their docs

**Pull request closed without merging**